### DISCUSSION TITLE
Purpose of <Field /> component?

### DISCUSSION THREAD

**Discussion by SpLouk** [tailwindlabs/headlessui#3397] [Help]
I noticed that the code snippets for headless-ui components all include the <Field /> component wrapping the main UI component. I can't find any documentation for what the Field component is for. Can someone please tell me what purpose it serves? It might be a good idea for the documentation to be updated also.

**Comment by msnegurski** [comment]
From what I see, it ads aria-labelledby and aria-describedby to corresponding label/description, also makes clicking on label to focus on input and other small stuff like that
